docuware-connector/config/rest-clients.yaml

```diff
@@ -17,6 +17,7 @@ RestClients:
       http.protocol.handle-redirects: 'true'
       JSON.Serialization.PROPERTY_INCLUSION: NON_NULL
       jersey.config.client.connectTimeout: ${ivy.var.docuware-connector.connectTimeout}
+      Organization: ${ivy.var.docuware-connector.organization}
```

Comment:
I would keep the domain specific properties together; so move it up after the 'LogonUrl'.
Then follow the technical properties, fine-tunning the rest-client...

Reply:
Thank you, I got your idea & would move it up right away 👍